include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -292,6 +292,42 @@ def TTIR_MatmulOp : TTIR_DPSOp<"matmul"> {
 }
 // ANCHOR_END: adding_an_op_matmul_ttir
 
+
+def TTIR_Conv2dOp : TTIR_DPSOp<"conv2d"> {
+    let summary = "Conv2d operation.";
+    let description = [{
+      Conv2d operation.
+    }];
+
+    let arguments = (ins AnyRankedTensor:$input,
+                         AnyRankedTensor:$weight,
+                         AnyRankedTensor:$bias,
+                         AnyRankedTensor:$output,
+                         SI32Attr:$kernel_height,
+                         SI32Attr:$kernel_width,
+                         SI32Attr:$stride_height,
+                         SI32Attr:$stride_width,
+                         SI32Attr:$dilation,
```

Comment:
Single dilation whereas in TTNN ops it's divided into vertical and horizontal dilation. Is there a reason not to standardize this? Preferably have the two values, to match the style of other attrs (stride, kernel size, etc.).

Reply:
AS of right now, forge only has this single dilation attribute, so in TTIR I keep the single one and in TTNN I use that same value for both the vertical and horizontal dilation.

Reply:
We can change things in the forge, right?

Reply:
I understand. However, it seems wrong for one dialect to have less expressive power than the other. Forge can easily adapt by passing the same value twice if they choose to have 1 dilation value in their APIs.